download_jartic_archives.py

```
@@ -0,0 +1,127 @@
+                        cached_count += 1
+                    else:
+                        tqdm.write(f"📥 Downloading {month_str}...")
+                        # The download_archive method will show its own progress bar
+                        await downloader.download_archive(year, month)
+                        downloaded_count += 1
+                        file_size_mb = archive_path.stat().st_size / (1024 * 1024)
+                        tqdm.write(f"✅ Downloaded {month_str} ({file_size_mb:.1f} MB)")
+                    
+                    overall_progress.update(1)
+                    
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)